stix_shifter_modules/gcp_chronicle/stix_transmission/results_connector.py

```diff
@@ -0,0 +1,374 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_results_connector import BaseResultsConnector
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from stix_shifter_utils.utils import logger
+import json
+from httplib2 import ServerNotFoundError
+from google.auth.exceptions import RefreshError
+import copy
+import re
+
+
+class ResultsConnector(BaseResultsConnector):
+    EMAIL_PATTERN = re.compile(r'(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)')
+
+    def __init__(self, api_client):
+        self.api_client = api_client
+        self.logger = logger.set_logger(__name__)
+        self.connector = __name__.split('.')[1]
+
+    def create_results_connection(self, search_id, offset, length):
+        """
+        Fetching the results using search id, offset and length
+        :param search_id: str, search id generated in transmit query
+        :param offset: str, offset value
+        :param length: str, length value
+        :return: dict
+        """
+        return_obj = {}
+        response_dict = {}
+        result = []
+        next_page_token = '0'
+        result_count = 0
+        page_size = self.api_client.result_limit
```

Comment:
why does the `page_size` set  to `result_limit`? shouldn't this be set to `length`?

Reply:
As discussed, updated the code to set the page_size with total of length and offset.